libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -55,8 +55,8 @@ internal class MapboxTripSession(
 ) : TripSession {
 
     companion object {
-        internal const val UNCONDITIONAL_STATUS_POLLING_PATIENCE = 2000L
-        internal const val UNCONDITIONAL_STATUS_POLLING_INTERVAL = 1000L
+        internal var UNCONDITIONAL_STATUS_POLLING_PATIENCE = 2000L
```

Comment:
in a second round noted: `UNCONDITIONAL_STATUS_POLLING_PATIENCE` and `UNCONDITIONAL_STATUS_POLLING_INTERVAL` are used in `MapboxTripSession` out of UI thread and outside developers don't have access to it. I'd say both values must be marked as `@Volatile`

Reply:
Good catch!